app/controllers/region_admin/donors_controller.rb

```diff
@@ -0,0 +1,36 @@
+class RegionAdmin::DonorsController < ApplicationController
```

Comment:
I've always favored the nested syntax to keep the nesting and constant lookup predictable. Not sure if it's an issue here since we most likely won't be referencing any other constants in the `RegionAdmin` module, but I'd have a slight preference toward breaking it apart:

```ruby
module RegionAdmin
  class DonorsController < ApplicationController
    ...
  end
end
```